### PR TITLE
Merging HinglishBot App into Dev Branch

### PR DESCRIPTION
**Pull request by radhikasethi2011**
- requirements.txt of hinglishbot (will be updated)

### CODE REVIEW

File: twitterbot.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        
+    
+
+def follower_count()->list:
+    followers = twitter_API.followers()
+    return (followers)
+
+num = []
+num = follower_count()
+mention_followers(num)
```
Comment (meghanabhange) [inline twitterbot.py:43]: We can remove the mention followers part for now in this part? 
We will use this in a different workflow? Like for now, we can use tweepy to get the full_text of the tweet from the URL?

File: twitterbot.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+st.image(
+    image="https://verloop.io/wp-content/uploads/2020/08/cropped-VP.io-Website-Grey@2x.png"
+)
+
+
+preview_count = 9
+warning_count = 10000
+
+tweet_url = st.text_input('Enter Tweet URL', 'enter a tweet url here, eg - https://twitter.com/Twitter/status/1320822556614676480')
+sentiment = mood(tweet_url)
```
Comment (meghanabhange) [inline twitterbot.py:18]: Maybe instead of passing the tweet URL, we can pass the tweet text, that would be helpful right?

**Review by meghanabhange — CHANGES_REQUESTED** [review]

File: twitterbot.py (hunk 1 above)
Comment (radhikasethi2011) [reply twitterbot.py:43]: Agreed. Okay

File: twitterbot.py (hunk 2 above)
Comment (radhikasethi2011) [reply twitterbot.py:18]: good catch, thanks. On it

**Comment by radhikasethi2011** [comment]
Rebased with secrets-temp branch @meghanabhange

**Review by meghanabhange — APPROVED** [review]

**Pull request merged**